devsetup/standalone/openstack.sh

```diff
@@ -64,6 +64,14 @@ parameter_defaults:
   NovaComputeLibvirtType: qemu
   ValidateGatewaysIcmp: false
   ValidateControllersIcmp: false
+  OctaviaAmphoraSshKeyFile: /root/.ssh/id_ecdsa.pub
+  OctaviaGenerateCerts: true
+  OctaviaLogOffload: true
+  OctaviaForwardAllLogs: true
+  StandaloneExtraConfig:
+    nova::compute::libvirt::services::libvirt_virt_type: kvm
+    nova::compute::libvirt::virt_type: kvm
```

Comment:
Octavia requires nested virtualization, so I think it's fine to have it in this PR,
that said, on line 64 we have `NovaComputeLibvirtType: qemu` which contradicts these 2 options.

Reply:
```
$ git grep NovaComputeLibvirtType
deployment/nova/nova-modular-libvirt-container-puppet.yaml:  NovaComputeLibvirtType:
deployment/nova/nova-modular-libvirt-container-puppet.yaml:            nova::compute::libvirt::services::libvirt_virt_type: {get_param: NovaComputeLibvirtType}
deployment/nova/nova-modular-libvirt-container-puppet.yaml:            nova::compute::libvirt::virt_type: {get_param: NovaComputeLibvirtType}
```

I think that `NovaComputeLibvirtType: kvm` should be enough

Reply:
Thanks. I removed the line because kvm is the [default](https://opendev.org/openstack/tripleo-heat-templates/src/branch/master/deployment/deprecated/nova/nova-libvirt-container-puppet.yaml#L187)

I can set it explicitly if you prefer, however.